lib/phoenix_live_view.ex

```diff
@@ -1003,6 +1003,27 @@ defmodule Phoenix.LiveView do
     Phoenix.LiveView.Channel.send_update(module, id, assigns)
   end
 
+  @doc """
+  Similar to `send_update/2` but the update will be delayed according to the given `time_in_milliseconds`.
+
+  ## Examples
+
+      def handle_event("cancel-order", _, socket) do
+        ...
+        send_update_after(Cart, [id: "cart", status: "cancelled"], 3000)
+        {:noreply, socket}
+      end
+  """
+  def send_update_after(module, assigns, time_in_milliseconds) do
```

Comment:
Can you please add guards for all 3 arguments? Then we should be ready to go!

Reply:
done!